src/views/index.twig

```diff
@@ -72,13 +72,12 @@
         </div>
 
         <div style="max-width: 38rem" class="mx-auto pt-4 pb-2 animated fadeInUp">
```

Comment:
@kimpenguin @abourne1 the width of the landing page content can be adjusted here to make sure the text doesn't break onto a new line, it's just an inline style.

Reply:
upped the max-width